src/test/java/edu/unc/lib/boxc/migration/cdm/PermissionsCommandIT.java

```diff
@@ -0,0 +1,119 @@
+package edu.unc.lib.boxc.migration.cdm;
+
+import edu.unc.lib.boxc.migration.cdm.model.PermissionsInfo;
+import edu.unc.lib.boxc.migration.cdm.services.PermissionsService;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class PermissionsCommandIT extends AbstractCommandIT {
+    @BeforeEach
+    public void setup() throws Exception {
+        initProjectAndHelper();
+        setupChompbConfig();
+    }
+
+    @Test
+    public void generateDefaultPermissions() throws Exception {
+        String[] args = new String[] {
+                "-w", project.getProjectPath().toString(),
+                "permissions", "generate",
+                "-id", "default",
+                "--everyone", "canViewMetadata",
+                "--authenticated", "canViewMetadata"};
+        executeExpectSuccess(args);
+        assertDefaultMapping("default", "canViewMetadata",
+                "canViewMetadata");
+    }
+
+    @Test
+    public void generateDefaultPermissionsNone() throws Exception {
```

Comment:
To avoid confusion, it'd be good to name this something like `generateDefaultPermissionsUnspecified` since "None" is a specific access control setting (as demonstrated in the staff only test)